src/mcode-pilot/services/outcomes/CLQOutcomesService.js

```diff
@@ -105,7 +107,7 @@ export default class CLQOutcomesService extends IOutcomesService {
                 code: code.code.code,
                 codeSystem: code.codeSystem.uri,
                 displayName: code.displayText.string,
-                value: value
+                value: _.startCase(_.toLower(value))
```

Comment:
Not a big deal but is the `toLower` call here necessary? From a glance at the docs it seems like  `startCase(toLower(x))` should === `startCase(x)`

Reply:
I'll take a look at the docs but I thought that startCase only capitalize the first letter and left the rest of the string alone.

Reply:
Some simple tests on strings and startCase will ensure that every word in the string is capitalized and leave the rest of the string alone.  So "HEY THAR" will stay "HEY THAR".

Reply:
Yeah I just tried it too. startCase is weird.